config/charts/inferencepool/templates/epp-config.yaml

```diff
@@ -39,16 +39,11 @@ data:
     - name: predicted-latency-prefix
       plugins:
       - pluginRef: prefix-cache-scorer
-    - name: predicted-latency-no-routing
```

Comment:
Do we still need the `predicted-latency-prefix` profile? I thought preparedata plugin will allow us to get rid of it.